java/src/main/java/com/google/oak/transparency/RekorLogEntry.java

```diff
@@ -41,31 +44,38 @@ private RekorLogEntry(LogEntry logEntry) {
   // The following nested classes represent a subset of Rekor types defined in
   // <https://github.com/sigstore/rekor/tree/2978cdc26fdf8f5bfede8459afd9735f0f231a2a/pkg/generated/models>.
   //
-  // These classes are intentionally made package-private and immutable, as the clients are not
-  // expected to instantiate them directly. The fields are not explicitly made final to allow
+  // These classes are intentionally made package-private and immutable, as the
+  // clients are not
+  // expected to instantiate them directly. The fields are not explicitly made
+  // final to allow
   // instantiation with Gson.
   //
 
   /**
    * Represents a Rekor LogEntry.
    *
-   * <p>Based on
+   * <p>
+   * Based on
    * <https://github.com/sigstore/rekor/blob/2978cdc26fdf8f5bfede8459afd9735f0f231a2a/pkg/generated/models/log_entry.go#L89.>
    */
   static class LogEntry {
-    /** We cannot directly use the type `Body` here, since body is Base64-encoded. */
+    /**
+     * We cannot directly use the type `Body` here, since body is Base64-encoded.
+     */
     String body;
 
     /**
-     * Unmarshaled body of this LogEntry. It is declared as a transient field, so that it is
+     * Unmarshaled body of this LogEntry. It is declared as a transient field, so
+     * that it is
```

Comment:
There seems to be some strange formatting going on in a few place.

Reply:
Thanks for noticing this. Should be fixed.